dist/index.html

```diff
@@ -40,15 +42,15 @@ <h1 class="welcome-user">Welcome, User You spent $0, On Rooms So Far</h1>
         <input type="date" id="calendar-start" name="trip-start" min="2022-01-01" max="2022-12-31" required>
         <input type="submit" id="calendarCheckInBtn">
 
-          <label for="selecting-room">Filter Room Type:</label>
-          <select name="room-types" id="room-types">
-            <option>Room Type</option>
-            <option>Residential Suite</option>
-            <option>Junior Suite</option>
-            <option>Suite</option>
-            <option>Single Room</option>
-          </select>
-          <input type="submit" id="dropDownBtn">
+        <label for="selecting-room">Filter Room Type:</label>
+        <select name="room-types" id="room-types">
+          <option>Room Type</option>
+          <option>Residential Suite</option>
+          <option>Junior Suite</option>
+          <option>Suite</option>
+          <option>Single Room</option>
+        </select>
```

Comment:
Nice work here! If you have time, make sure you can 'tab' through your whole page. You might need to add some stuff to your <option> elements like `tabindex="0" ` to make it easier to naviagate.